homeassistant/components/holiday/config_flow.py

```diff
@@ -30,6 +31,32 @@
 SUPPORTED_COUNTRIES = list_supported_countries(include_aliases=False)
 
 
+def get_optional_provinces(country: str) -> list[Any]:
+    """Return the country provinces (territories).
+
+    Some territories can have extra or different holidays
+    from another within the same country.
+    Some territories can have different names (aliases).
+    """
+    province_options: list[Any] = []
+
+    if provinces := SUPPORTED_COUNTRIES[country]:
+        country_data = country_holidays(country, years=dt_util.utcnow().year)
+        if country_data.subdivisions_aliases and (
+            subdiv_aliases := country_data.get_subdivision_aliases()
+        ):
+            # Remember the user choice among all of the aliases available
+            province_options = [
+                SelectOptionDict(value=alias, label=alias)
```

Comment:
Both value and label is the same alias so then we don't need to use a `SelectOptionDict`

Reply:
Made obsolete by joining aliases as you did in workday.